storage/tsdb/tsdb.go

```diff
@@ -155,7 +155,7 @@ func Open(path string, l log.Logger, r prometheus.Registerer, opts *Options) (*t
 	// Initialize V3IO Adapter
 	cfgpath := os.Getenv("V3IO_FILE_PATH")
 	if cfgpath == "" {
-		cfgpath = "v3io.yaml"
+		cfgpath = "/etc/v3io/v3io.yaml"
```

Comment:
IMHO, we should change the name of default configuration file to `/etc/v3io/v3io-tsdb.yaml` to distinguish from the other config files